pkg/daemon/server/service/rater/pod_tracker.go

```diff
@@ -90,8 +90,10 @@ func (pt *PodTracker) Start(ctx context.Context) error {
 					var vType string
 					if v.IsReduceUDF() {
 						vType = "reduce"
+					} else if v.IsASource() {
+						vType = "source"
 					} else {
-						vType = "non_reduce"
+						vType = "other"
```

Comment:
i think `non_reduce` is a good word, as it clears highlights the exclusion list without any ambiguity :)

Reply:
I thought about rename it to `non_reduce_non_source` but it seems a bit too long to me. :)